hca/upload/s3_agent.py

```diff
@@ -83,7 +83,8 @@ def copy_s3_file(self, s3_path, target_bucket, target_key, content_type, report_
             'ExtraArgs': {
                 'ContentType': content_type,
                 'MetadataDirective': 'REPLACE',
-                'ACL': 'bucket-owner-full-control'
+                'ACL': 'bucket-owner-full-control',
+                'Metadata': checksums
```

Comment:
Does this stick in s3? I remember I had some difficulties earlier with s3 metadata

Reply:
Yep! Tried and tested!